remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/indexing/search5/Search5Annotations.java

```diff
@@ -31,6 +31,36 @@ public final class Search5Annotations {
    public static final String STORE_YES = "Store.YES";
    public static final String STORE_NO = "Store.NO";
 
+   public static final String PROJECTABLE_DEFAULT = "Projectable.DEFAULT";  // translates to either YES or NO depending on the indexing backend
+   public static final String PROJECTABLE_NO = "Projectable.NO";
```

Comment:
Do we really need to add stuff to the HS 5 compatibility layer ?

Reply:
These rows will be overwritten by further commit messages, when we support the new annotations (point 2).

Reply:
I removed with the last push. Basically I used this legacy code to have a working solution to replace with the proper final one.